.github/workflows/deploy.yml

```
@@ -15,7 +15,7 @@ on:
         description: "Dry run — validate config & print plan, skip register/deploy"
         type: boolean
         default: false
-
+b
```

actionlint (1.7.11)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)